src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -353,8 +372,6 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    help='Determines whether the application should be recreated on update. If value=true, the rest of the upgrade policy parameters are not allowed.')
         c.argument('upgrade_replica_set_check_timeout', options_list=['--upgrade-replica-set-check-timeout', '--replica-check-timeout', '--rep-check-timeout'],
                    help='Specify the maximum time, in seconds, that Service Fabric waits for a service to reconfigure into a safe state, if not already in a safe state, before Service Fabric proceeds with the upgrade.')
-        c.argument('instance_close_delay_duration', options_list=['--instance-close-delay-duration', '--instance-close-duration', '--close-duration'],
```

Comment:
Only the instance_close_duration in service needs to be removed, not the application one